packages/react/src/components/Button/Button.tsx

```diff
@@ -21,66 +18,55 @@ export type ButtonProps = {
    */
   size?: 'sm' | 'md' | 'lg';
   /**
-   * If `Button` should fill full width of its container
+   * If `Button` should fill width of its container
    * @default false
    */
-  fullWidth?: boolean;
-  /** Toggle icon only styling, pass icon as children
+  fill?: boolean;
```

Comment:
I'm not sure about this, it could easily be interpeted as a "fill button", which in a lot of cases means solid color.

Reply:
Enig - kanskje `full` er et bedre ord? `fill` var valgt siden dette følger `grid` og `object-fit` sin navngivning :)

Reply:
Suggest we remove for now to provide better flexibility, alternatively we can add it back if needed.

Reply:
This is used several places, so keeping for now :)